.github/workflows/release.yml

```diff
@@ -78,5 +78,5 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           GITHUB_REPOSITORY: ${{ github.repository }}
-          GITHUB_REF: ${{ github.ref }}
+          GITHUB_REF: ${{ github.sha }}
```

Comment:
Consider using the `${{ github.sha }}` instead of `${{ github.ref }}` to target the correct commit for the release. This makes the workflow more reliable and robust. [important]